examples/async/.babelrc.js

```diff
@@ -1,11 +1,3 @@
 module.exports = {
-  plugins: [
-    [
-      '@babel/plugin-transform-runtime',
-      {
-        regenerator: true,
-      },
-    ],
-  ],
-  presets: ['@babel/preset-env'],
+  presets: [['@babel/preset-env', {targets: {node: 'current'}}]],
```

Comment:
```suggestion
  // See https://babeljs.io/docs/en/babel-preset-env#targets
  presets: [['@babel/preset-env', {targets: {node: 'current'}}]],
```

or something like that? Not for us, but for people reading our examples

Reply:
maybe not though. `node: 'current'` is _always_ correct when running in Jest, at least up until we support browsers

Reply:
maybe switch to Node 6 which doesn't have async/await natively?

Reply:
Nah, when you run tests, you should transform as little as possible. And we run our CI using node 6 so we verify that this config does work